### PR TITLE
Completed Skins Compendium Update

### PR DESCRIPTION
**Pull request by RockRoller01**
Updated this after 2 months. Everything SHOULD be ok.

### CODE REVIEW

File: wiki/Skinning/Compendium/en.md [hunk 1]
```diff
@@ -129,88 +156,123 @@ Compendium of all completed osu! skins.
...
+- [NeOsu 5 (NeOsu Surge)](https://osu.ppy.sh/community/forums/topics/202838) by [OsuMe65](https://osu.ppy.sh/users/852867)![][o!s]![][o!t]![][o!c]![][o!m]
 - [Neo Street](https://osu.ppy.sh/community/forums/topics/332576) by [OsuMe65](https://osu.ppy.sh/users/852867)![][o!s]![][o!m]
+- [Nero](https://osu.ppy.sh/community/forums/topics/651827) by [dlwnstns99](https://osu.ppy.sh/users/4585661)![][o!s]
 - [Nexus Slate & Ivory](https://osu.ppy.sh/community/forums/topics/55948) by [Xiao](https://osu.ppy.sh/users/17894)![][o!s]
+- [Nexus](https://osu.ppy.sh/community/forums/topics/432071) by [atturbo555](https://osu.ppy.sh/users/5730417)![][o!s]![][o!t]![][o!c]
 - [Nikaido Yuzu](https://osu.ppy.sh/community/forums/topics/604971) by [Lathler](https://osu.ppy.sh/users/9039378)![][o!s]
+- [NOEL](https://osu.ppy.sh/community/forums/topics/651827) by [dlwnstns99](https://osu.ppy.sh/users/4585661)![][o!s]
 - [Northen Ocean Hime](https://osu.ppy.sh/community/forums/topics/319797) by [-[Jess]-](https://osu.ppy.sh/users/4085133)![][o!s]![][o!c]
 - [Nui Harime](https://osu.ppy.sh/community/forums/topics/417954) by [carnationturtle](https://osu.ppy.sh/users/4461382)
```
Comment (Joehuu) [inline wiki/Skinning/Compendium/en.md]: Missing mode icons.

Comment (RockRoller01) [reply wiki/Skinning/Compendium/en.md]: Nope/missing on purpose. That skin simply doesn't have any modes skinned.

Comment (Joehuu) [reply wiki/Skinning/Compendium/en.md]: Really? There are no gameplay screenshots maybe that's why. I actually downloaded it to check. It skins osu!standard.

Says it on skin forum:
> Currently, this skin supports only osu!

Comment (RockRoller01) [reply wiki/Skinning/Compendium/en.md]: You're right. It's skinned, but ther are no screenshots. Fixed that.

File: wiki/Skinning/Compendium/en.md [hunk 2]
```diff
@@ -3,34 +3,43 @@
...
-# Compendium
-Compendium of all completed osu! skins.
+# Skins Compendium
+This is a compendium of all completed osu! skins
 
-- ![][o!s] means the skin contains osu!standard elements.
-- ![][o!c] means the skin contains osu!catch elements.
-- ![][o!t] means the skin contains osu!taiko elements.
-- ![][o!m] means the skin contains osu!mania elements.
+- ![][o!s] This means the skin contains osu!standard elements
```
Comment (TPGPL) [inline wiki/Skinning/Compendium/en.md]: `- ![][o!s] - skin contains osu!standard elements`

File: wiki/Skinning/Compendium/en.md [hunk 3]
```diff
@@ -3,34 +3,43 @@
...
+- ![][o!c] This means the skin contains osu!catch elements
```
Comment (TPGPL) [inline wiki/Skinning/Compendium/en.md]: `- ![][o!c] - skin contains osu!catch elements`

same for ones below

Comment (RockRoller01) [reply wiki/Skinning/Compendium/en.md]: Oh, I see. The phrasing and the heading changed for some reason. Looks like the programm for the compendium needs a bit of adjustment. I guess the old text is fine?

Comment (TPGPL) [reply wiki/Skinning/Compendium/en.md]: old text is fine for those, might actually add a hyphen if you want

File: wiki/Skinning/Compendium/en.md [hunk 4]
```diff
@@ -129,88 +156,123 @@ Compendium of all completed osu! skins.
...
+- [R4V3 F4L7](https://osu.ppy.sh/community/forums/topics/528744) by [atturbo555](https://osu.ppy.sh/users/5730417)![][o!s]![][o!t]![][o!c]![][o!m]
 
 ## S
 
 - [Sagiri Edition](https://osu.ppy.sh/community/forums/topics/588711) by [atturbo555](https://osu.ppy.sh/users/5730417)![][o!s]
+- [Sagiri](https://osu.ppy.sh/community/forums/topics/631673) by [atturbo555](https://osu.ppy.sh/users/5730417)![][o!s]
+- [Saikuus](https://osu.ppy.sh/community/forums/topics/620379) by [Saikkimus](https://osu.ppy.sh/users/9820272)![][o!s]![][o!t]![][o!c]![][o!m]
 - [Sakuya Izavoi](https://osu.ppy.sh/community/forums/topics/32589) by [MLGnom](https://osu.ppy.sh/users/46620)![][o!s]
+- [Sanoba Witch](https://osu.ppy.sh/community/forums/topics/654114) by [atturbo555](https://osu.ppy.sh/users/5730417)![][o!s]![][o!c]![][o!m]
 - [Senki Zesshou Symphogear](https://osu.ppy.sh/community/forums/topics/181715) by [Kurokoami](https://osu.ppy.sh/users/260933)![][o!s]![][o!t]![][o!c]![][o!m]
```
Comment (Shadowthegod) [inline wiki/Skinning/Compendium/en.md]: While you are at it, can you fix my name and remove the extra "o" after the second k? 🤔 👀

File: wiki/Skinning/Compendium/en.md (hunk 3 above)
Comment (RockRoller01) [reply wiki/Skinning/Compendium/en.md]: hyphen between `]` and `means`?

File: wiki/Skinning/Compendium/en.md (hunk 4 above)
Comment (RockRoller01) [reply wiki/Skinning/Compendium/en.md]: Oh sorry, will do that when I fix the other things

File: wiki/Skinning/Compendium/en.md (hunk 3 above)
Comment (TPGPL) [reply wiki/Skinning/Compendium/en.md]: yeah

File: wiki/Skinning/Compendium/en.md [hunk 5]
```diff
@@ -6,278 +6,368 @@
 # Compendium
 Compendium of all completed osu! skins.
 
-- ![][o!s] means the skin contains osu!standard elements.
-- ![][o!c] means the skin contains osu!catch elements.
-- ![][o!t] means the skin contains osu!taiko elements.
-- ![][o!m] means the skin contains osu!mania elements.
+- ![][o!s]  - means the skin contains osu!standard elements.
```
Comment (TPGPL) [inline wiki/Skinning/Compendium/en.md]: should be ` - `; too many spaces; same below

Comment (RockRoller01) [reply wiki/Skinning/Compendium/en.md]: I can change that later, but the german version has the same formatting iirc.

**Pull request merged**